src/Common/Help_Hub/Hub.php

```diff
@@ -0,0 +1,423 @@
+					'data' => [
+						'sysinfo_optin_nonce'        => wp_create_nonce( 'sysinfo_optin_nonce' ),
+						'clipboard_btn_text'         => _x( 'Copy to clipboard', 'Copy to clipboard button text.', 'tribe-common' ),
+						'clipboard_copied_text'      => _x( 'System info copied', 'Copy to clipboard success message', 'tribe-common' ),
+						'clipboard_fail_text'        => _x( 'Press "Cmd + C" to copy', 'Copy to clipboard instructions', 'tribe-common' ),
+						'sysinfo_error_message_text' => _x( 'Something has gone wrong!', 'Default error message for system info optin', 'tribe-common' ),
+						'sysinfo_error_code_text'    => _x( 'Code:', 'Error code label for system info optin', 'tribe-common' ),
+						'sysinfo_error_status_text'  => _x( 'Status:', 'Error status label for system info optin', 'tribe-common' ),
+					],
+				],
+			]
+		);
+
+		// Add the built-in accordion.
+		wp_enqueue_script( 'jquery-ui-accordion' );
+	}
+
+	/**
+	 * Get the telemetry opt in link.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_telemetry_opt_in_link(): string {
+		return add_query_arg(
+			[
+				'page'      => 'tec-events-settings',
+				'tab'       => 'general-debugging-tab',
+				'post_type' => 'tribe_events',
+			],
+			admin_url( 'edit.php' )
+		);
+	}
+
+	/**
+	 * Generates and outputs the iframe content if the correct parameters are provided.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function generate_iframe_content(): void {
+		$page   = tribe_get_request_var( 'page' );
+		$iframe = tribe_get_request_var( 'embedded_content' );
+
+		// Return early if $page is empty or not 'tec-events-help-hub', or if $iframe is empty.
+		if ( empty( $page ) || 'tec-events-help-hub' !== $page || empty( $iframe ) ) {
+			return;
+		}
+
+		// Enqueue our assets for the Iframe.
+		add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_help_page_iframe_assets' ] );
+
+		// Disable the admin bar for iframe requests.
+		// phpcs:ignore WordPressVIPMinimum.UserExperience.AdminBarRemoval.RemovalDetected
+		show_admin_bar( false );
+		// Render the iframe content.
+		$this->render_template(
+			'help-hub/support/iframe-content'
+		);
+
+		exit;
+	}
+
+	/**
+	 * Enqueues the necessary scripts for the help page and dequeues all theme styles.
+	 *
+	 * @since TBD
+	 * @return void
+	 */
+	public function enqueue_help_page_iframe_assets(): void {
+		define( 'IFRAME_REQUEST', true );
+		tribe_asset(
+			Tribe__Main::instance(),
+			'help-hub-iframe-style',
+			'help-hub-iframe.css',
+			null,
+			'wp_enqueue_scripts'
+		);
+		tribe_asset(
+			Tribe__Main::instance(),
+			'help-hub-iframe-js',
+			'admin/help-hub-iframe.js',
+			null,
+			'wp_enqueue_scripts',
+			[
+				'localize' => [
+					'name' => 'helpHubSettings',
+					'data' => [
+						'docsbot_key'    => $this->config->get( 'DOCSBOT_SUPPORT_KEY' ),
+						'zendeskChatKey' => $this->config->get( 'ZENDESK_CHAT_KEY' ),
+					],
+				],
+			]
+		);
+		$this->dequeue_theme_styles();
```

Comment:
Why do we need this?

Reply:
The generate_iframe_content() method renders a frontend-styled iframe within the Admin context rather than loading a traditional dashboard page. Since theme styles are designed for frontend use, they can affect the layout and styling of our Help Hub content. To ensure a consistent, unaltered appearance for the iframe, dequeue_theme_styles() removes these theme-specific styles.